homeassistant/components/surepetcare/__init__.py

```diff
@@ -185,3 +208,16 @@ async def async_update(self, arg: Any = None) -> None:
                 _LOGGER.error("Unable to retrieve data from surepetcare.io: %s", error)
 
         async_dispatcher_send(self.hass, TOPIC_UPDATE)
+
+    async def set_lock_state(self, flap_id: int, state: str) -> None:
+        """Update the lock state of a flap."""
+        if state.lower() == SureLockStateID.UNLOCKED.name.lower():
```

Comment:
Do the state value modifications in the service schema instead. There's `vol.Lower`.

Reply:
Added `vol.Lower`